### PR TITLE
ie bug

### PR DESCRIPTION
**Pull request by levibrown**
This is a fix for a ie9/10 that prevented the the src from switching on page load.  The solution was to set the image src to "null" before a source value is set.

### CODE REVIEW

File: src/picture_tag/img.coffee [hunk 1]
```diff
@@ -8,11 +8,12 @@ class @PictureTag.Img
     @$el = @_current()
     unless value == @$el.attr("src")
       @$el2 = @$el.clone()
+      @$el2.attr "src", null
```
Comment (jlsuttles) [inline src/picture_tag/img.coffee:11]: this is the whole, fix, right? so great.

Comment (levibrown) [inline src/picture_tag/img.coffee:11]: yep, that little bit took care of it.

**Comment by jlsuttles** [comment]
:star2:

**Comment by jlsuttles** [comment]
related https://github.com/G5/jquery-picture-tag/issues/1

**Pull request merged**